unsubscribe-automation/src/main.ts

```
@@ -0,0 +1,199 @@
+import { chromium, Page } from "playwright";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)